src/Cli/dotnet/Commands/Run/VirtualProjectBuildingCommand.cs

```diff
@@ -131,6 +129,23 @@ public VirtualProjectBuildingCommand(
     public bool NoBuild { get; init; }
     public string BuildTarget { get; init; } = "Build";
 
+    public ImmutableArray<CSharpDirective> Directives
+    {
+        get
+        {
+            if (field.IsDefault)
```

Comment:
❤️

Reply:
Ah, is this the new keyword for accessing the backing field of a property?